content/docs/latest/installing/cloud/azure.md

```diff
@@ -78,34 +76,66 @@ $ az vm create --name node-1 --resource-group group-1 --admin-username core --us
-    "urn": "kinvolk:flatcar-container-linux:alpha:2430.0.0",
-    "version": "2430.0.0"
-  }
-]
-$ az vm create --name node-1 --resource-group group-1 --admin-username core --user-data config.ign --image kinvolk:flatcar-container-linux:alpha:2430.0.0
+$ az vm image list --all -p kinvolk -f flatcar -s alpha-gen2 --query '[-1]'
+{
+  "architecture": "x64",
+  "offer": "flatcar-container-linux-free",
+  "publisher": "kinvolk",
+  "sku": "alpha-gen2",
+  "urn": "kinvolk:flatcar-container-linux-free:alpha-gen2:3874.0.0",
+  "version": "3874.0.0"
+}
+$ az vm create --name node-1 --resource-group group-1 --admin-username core --user-data config.ign --image kinvolk:flatcar-container-linux-free:alpha-gen2:3874.0.0
         </pre>
       </div>
     </div>
   </div>
 </div>
 
-You can use both image offers `flatcar-container-linux` and `flatcar-container-linux-free`, the contents are the same.
-The SKU, which is the third element of the image URN, relates to one of the release channels and also depends on whether to use HyperV Generation 1 or 2.
-Generation 1 instance types use the channel names `alpha`, `beta` or `stable` as is; for Generation 2 instance types please append `-gen2` to the channel name, i.e., use one of `alpha-gen2`, `beta-gen2` or `stable-gen2`.
-This means the Gen 2 image URN for the above example for a Stable release becomes `flatcar-container-linux:stable-gen2:2345.3.0`.
+Use the offer named `flatcar-container-linux-free`, there is also a legacy offer called `flatcar-container-linux` with the same contents.
+The SKU, which is the third element of the image URN, relates to one of the release channels and also depends on whether to use Hyper-V Generation 1 or 2 VM.
+Generation 2 instance use UEFI boot and should be preferred, the SKU matches the pattern `<channel>-gen`: `alpha-gen2`, `beta-gen2` or `stable-gen2`.
+For Generation 1 instance types drop the `-gen2` from the SKU: `alpha`, `beta` or `stable`.
+
+Before being able to use the offers, you may need to accept the legal terms once, here done for `flatcar-container-linux-free` and `stable-gen2`:
+
+```shell
+az vm image terms show --publish kinvolk --offer flatcar-container-linux-free --plan stable-gen2
+az vm image terms accept --publish kinvolk --offer flatcar-container-linux-free --plan stable-gen2
+```
 
+For quick tests the official Azure CLI also supports an alias for the latest Flatcar stable image:
+```shell
+az vm create --name node-1 --resource-group group-1 --admin-username --user-data config.ign --image FlatcarLinuxFreeGen2
+```
 
-Before being able to use them, you may need to accept the legal terms once, here done for `flatcar-container-linux` and `stable`:
+### CoreVM
 
+Flatcar images are also published under an offer called `flatcar-container-linux-corevm-amd64`. This offer does not require accepting image terms and does not require specifying plan information when creating instances or building derived images. The contents of the images matches the other offers.
```

Comment:
```suggestion
Flatcar images are also published under an offer called `flatcar-container-linux-corevm-amd64`. This offer does not require accepting image terms and does not require specifying plan information when creating instances or building derived images. The content of the images matches the other offers.
```

Reply:
This offer only has gen2, or? Maybe worth mentioning this

Reply:
no, for amd64 it has both

Reply:
fixed the word and switched the example to stable-gen2 explicitly

Reply:
The gen1 is really just `stable` and the `-s` flag would thus return both gen1 and gen2 if run with `stable` as value. This means there is no good way of just listing gen1 images, maybe we should call this out.

Reply:
That also affects the `-free` offer